closed/OpenJ9.gmk

```diff
@@ -55,6 +55,13 @@ NUMCPU := $(shell grep -c ^processor /proc/cpuinfo)
 #$(info NUMCPU = $(NUMCPU))
 override MAKEFLAGS := -j $(NUMCPU)
 
+# honour the --disable-warnings-as-errors configure parameter during compilation of OMR
+ifeq (false,$(WARNINGS_AS_ERRORS))
+  OMR_EXTRA_CONFIGURE_ARGS := --disable-warnings-as-errors
+else
+  OMR_EXTRA_CONFIGURE_ARGS := --enable-warnings-as-errors
```

Comment:
I believe that enable warnings as errors is an OMR default.  The else path could be left off.

Reply:
I'm leaving that so that if the OMR default is changed, the global flag is still honoured.